src/Sarif.Viewer.VisualStudio.Core/Models/SuppressionCollection.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.ObjectModel;
+
+namespace Microsoft.Sarif.Viewer.Models
+{
+    internal class SuppressionCollection : ObservableCollection<SuppressionModel>
+    {
+        public SuppressionCollection()
```

Comment:
>SuppressionCollection [](http://example.com/codeflow?start=15&length=21)

do u need this?

Reply:
I would also rename to SuppressionModelCollection

Reply:
or remove this entire class and use like the others
var suppressions = new ObservableCollection<SuppressionModel>();

Reply:
removed. it was original created for data binding in SARIF explorer suppression tab. since the design changed not needed any more